pkg/assume/assume.go

```diff
@@ -75,7 +75,17 @@ func AssumeCommand(c *cli.Context) error {
 
 	if profile == nil {
 
+		//load config to check frecency enabled
+		cfg, err := config.Load()
+		if err != nil {
+			return err
+		}
+
 		fr, profiles := awsProfiles.GetFrecentProfiles()
+
+		if cfg.Ordering == "Alphabetical" {
+			profiles = awsProfiles.GetAlphabeticalProfiles()
```

Comment:
I'm thinking that this should just call awsProfiles.ProfileNames()

Rather than creating a new wrapper method here.